### ISSUE TITLE
asciidoctor-epub3 doesn't handle absolute image paths

### ISSUE THREAD

**Issue by slonopotamus** [asciidoctor/asciidoctor-epub3#199]
main.adoc:

```
= Book
:doctype: book
:imagesdir: images

include::_chapter1.adoc[leveloffset=+1]
```

chapter.adoc:

```
= Title

image::/usr/share/asciidoc/images/icons/home.png[]
```

When trying to create EPUB from this, asciidoc-epub3 complains: `asciidoctor: ERROR: _content.adoc: image not found or not readable: <basedir>/images/usr/share/asciidoc/images/icons/home.png`

Note that path was converted from absolute into relative. This bug prevents asciidoctor-epub3 from implementing a fix similar to asciidoctor/asciidoctor-pdf#1034.

This bug is not present in asciidoc-pdf.

The bug is not present if spine document doesn't define `:imagesdir:`, however generated EPUB doesn't show the image.

**Comment by slonopotamus** [comment]
Testcase project: https://github.com/slonopotamus/asciidoc-epub3-absolute-image

**Comment by slonopotamus** [comment]
Actually, this is a duplicate of #190, just with a simpler way to reproduce.

**Comment by mojavelinux** [comment]
This would be a really nice scenario to use to try to improve the test framework. The easier it is to set up a scenario in the test suite, the more tests will be written. That's exactly what happened with Asciidoctor PDF.

**Comment by slonopotamus** [comment]
I'm actually now able to repro and fix #190 without adding support for absolute image paths because fix doesn't involve using `:data-uri:`.
